plugins/wasd_controls.js

```diff
@@ -86,14 +93,21 @@ BBPlugin.register('wasd_controls', {
 			type: 'number',
 			value: 100,
 			min: 1
-		}))
+		}));
 
 		deletables.push(new Setting('wasd_y_level', {
 			name: 'WASD Navigation at Y Level',
 			description: 'Navigate using WASD at consistent Y level rather than on camera plane',
 			category: 'preview',
 			value: true
-		}))
+		}));
+
+		deletables.push(new Setting('requires_hold_right_mouse', {
```

Comment:
Settings are global. The setting name and ID do not make sense outside of the context of this plugin. The context needs to be included, like it is in the other setting.

Reply:
I'm a bit confused, you mean the name of the setting? If that's the case, is 'wasd_requires_hold_right_mouse' a better name?